### PR TITLE
Hyva themes

### PR DESCRIPTION
**Pull request by MaximGns**
**Issue:** closes #

**Similarweb rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by MaximGns** [comment]
Issue: https://github.com/simple-icons/simple-icons/issues/8038

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5227,6 +5227,11 @@
             "source": "https://en.wikipedia.org/wiki/File:Hyundai_Motor_Company_logo.svg",
             "guidelines": "https://www.hyundai.pl/fileadmin/user_upload/media/logo/201607_HYU_Guideline_ENG_small.pdf"
         },
+        {
+            "title": "Hyva Themes",
+            "hex": "0A23B9",
+            "source": "https://drive.google.com/file/d/1ohplW2WhEdAo8z5daUpFxvvtuJG56UsS/view"
```
Comment (barryvdh) [inline _data/simple-icons.json:5233]: Perhaps better to use https://hyva.io/ https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines

**Comment by barryvdh** [comment]
This is the current SVG:
https://shields.io/badge/Hyv%C3%A4_Theme-Compatible---?style=for-the-badge&labelColor=F6F7FF&color=0B23B9&logoColor=0B23B9&logo=data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0iTTkuNjI1IDkuNzdjMS4zNTUtMS4wNDcgMy4wMDgtMS43MDcgNC44NjctMS43MDcgMy44MDUgMCA3LjUwOCAyLjM2NyA3LjUwOCA4LjQzM1YyNGgtNC44Njd2LTcuNTA0YzAtMy4xMjktMS41MjgtNC4zMDgtMy42MTMtNC4zMDgtMS43MjMgMC0yLjk3My45Ni0zLjY2OCAxLjk3NlYyNEg0Ljk2NVY0LjUzMUgyTDMuNTYzIDBoNi4wNjJ2OS43N3pNMTYuNjQgMEgyMmwtMS41NyA0LjUzMWgtNS4zNTJMMTYuNjQgMHoiLz48L3N2Zz4K
image

Maybe it would be better to fill the dot?

https://shields.io/badge/Hyv%C3%A4_Theme-Compatible---?style=for-the-badge&labelColor=F6F7FF&color=0B23B9&logoColor=0B23B9&logo=data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0iTTkuNjI1IDkuNzdjMS4zNTUtMS4wNDcgMy4wMDgtMS43MDcgNC44NjctMS43MDcgMy44MDUgMCA3LjUwOCAyLjM2NyA3LjUwOCA4LjQzM1YyNGgtNC44Njd2LTcuNTA0YzAtMy4xMjktMS41MjgtNC4zMDgtMy42MTMtNC4zMDgtMS43MjMgMC0yLjk3My45Ni0zLjY2OCAxLjk3NlYyNEg0Ljk2NVY0LjUzMUgyTDMuNTYzIDBoNi4wNjJ2OS43N3pNMTYuNjQgMEgyMmwtMS41NyA0LjUzMWgtNS4zNTJMMTYuNjQgMHoiLz48L3N2Zz4K
image

```svg
<svg role="img" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M9.625 9.77c1.355-1.047 3.008-1.707 4.867-1.707 3.805 0 7.508 2.367 7.508 8.433V24h-4.867v-7.504c0-3.129-1.528-4.308-3.613-4.308-1.723 0-2.973.96-3.668 1.976V24H4.965V4.531H2L3.563 0h6.062v9.77zM16.64 0H22l-1.57 4.531h-5.352L16.64 0z"/></svg>
```

**Review by barryvdh — APPROVED** [review]

**Comment by MaximGns** [comment]
@LitoMore Is there something more needed?

**Comment by adamrusted** [comment]
Unfortunately, I'm going to have to close this as hyva.io does not meet the criteria for inclusion within Simple Icons (it's current SimilarWeb rank is [1,195,631](https://www.similarweb.com/website/hyva.io/#overview). Please feel free to re-open this, and the attributed issue, should it reach our metrics defined [here](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-an-icon).

**Pull request closed without merging**